validator/client/runner.go

```diff
@@ -67,7 +67,7 @@ func run(ctx context.Context, v iface.Validator) {
 			}
 		}
 	} else {
-		log.Warnln("Validator client started without proposer settings such as fee recipient" +
+		log.WithError(err).Warnln("Validator client started without proposer settings such as fee recipient" +
```

Comment:
err will be nil here always.

```suggestion
		log.Warn("Validator client started without proposer settings such as fee recipient" +
```